cmake/SfizzConfig.cmake

```diff
@@ -45,7 +45,11 @@ else()
     find_package(PkgConfig REQUIRED)
     pkg_check_modules(SNDFILE "sndfile" REQUIRED)
     target_include_directories(sfizz-sndfile INTERFACE ${SNDFILE_INCLUDE_DIRS})
-    target_link_libraries(sfizz-sndfile INTERFACE ${SNDFILE_LIBRARIES})
+    if (SFIZZ_STATIC_LIBSNDFILE)
+        target_link_libraries(sfizz-sndfile INTERFACE ${SNDFILE_STATIC_LDFLAGS})
```

Comment:
CMake's documention indicates there should exist a variable `SNDFILE_STATIC_LIBRARIES`.
Perhaps this was meant instead of `LDFLAGS`?

Reply:
Yes, I remember using STATIC_LIBRARIES and switching to LDFLAGS for some reason, but I think it was because I had the libraries in a non-standard location maybe.

Using STATIC_LIBRARIES seems to work now so I switched it back.